### ISSUE TITLE
Loading skeleton for the item count 

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#5893]
https://github.com/kodadot/nft-gallery/assets/90852205/cb666ec5-f42a-48fa-93ae-e58a33af42ee

so right now, after #5872 the preview always jumps when the actual content is loaded. 
Thus it would be nice to have loading skeleton for the item count as well. 

good follow up for @leo-anderson-x

**Comment by leo-anderson-x** [comment]
👋

**Comment by kodabot** [comment]
ASSIGNED - @leo-anderson-x 🔒 LOCKED -> Thursday, May 11th 2023, 13:59:32 UTC -> 36 hours
